src/pages/Authorization/components/SignInForm/SignInForm.tsx

```diff
@@ -1,56 +1,68 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Link, useHistory } from 'react-router-dom';
+import { useAppDispatch, useAppSelector, messages } from 'shared';
 import { Formik } from 'formik';
 import {
   Button, Title, Footer,
-} from '../../components';
+} from 'components';
 import { ValidationForSignInForm } from './ValidationForSignInForm';
-import { SIGN_UP_PATH, RESTORE_PASSWORD_PATH, PATIENT_VIEW_PATH } from '../../routes/routes';
+import { loginUser, getStatus } from '../../redux';
+import { SIGN_UP_PATH, RESTORE_PASSWORD_PATH, PATIENT_VIEW_PATH } from 'routes/routes';
 import { FormStyled, InputEmail, InputPassword } from './SignInForm.styles';
-import { messages } from '../../shared';
+import { SignInFormTypes } from 'pages';
 
 export function SignInForm() {
   const { push } = useHistory();
+  const dispatch = useAppDispatch();
+  const loginStatus = useAppSelector(getStatus);
+
+  useEffect(() => {
+    if (loginStatus === 'OK') {
+      push(PATIENT_VIEW_PATH);
+    }
+  }, [push, loginStatus]);
+
 
   const initialValues = {
     email: '',
     password: '',
   };
 
-  function handleClick() {
-    push(PATIENT_VIEW_PATH);
-  }
+  const submitSignInForm = async (values: SignInFormTypes) => {
+    const userData = {
+      userName: values.email,
+      password: values.password,
+    };
+    await dispatch(loginUser(userData));
+  };
 
   return (
     <Formik
       initialValues={initialValues}
       validationSchema={ValidationForSignInForm}
-      onSubmit={(values, { setSubmitting }) => {
-        console.log(values);
-        setSubmitting(false);
-        handleClick();
-      }}
+      onSubmit={submitSignInForm}
     >
-      {() => (
-        <FormStyled>
+      {({ handleSubmit, isValid, dirty }) => (
+        <FormStyled onSubmit={handleSubmit}>
           <Title>
             <p>Sign In</p>
           </Title>
           <InputEmail
             label="Email"
             name="email"
             type="email"
-            placeholder="bart@gmail.com"
+            placeholder="email@gmail.com"
```

Comment:
u should not use gmail com in the placeholder

Reply:
I renaimed it to 'Email'